### PR TITLE
bookings controller complete, with new and edit html forms

### PR DESCRIPTION
**Pull request by tim94white**
bookings controller completed
new booking form complete
edit booking form complete

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,2 +1,50 @@
 class BookingsController < ApplicationController
+
+  def index
+    @bookings = Booking.all
+  end 
+
+  def new
+    @booking = Booking.new
+    @furnishing = Furnishing.find(params[:id])
```
Comment (Chloebm) [inline app/controllers/bookings_controller.rb]: remove this line please!

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+  end 
+  
+  def create
+    @booking = Booking.new(booking_params)
+    # @furnishing = Furnishing.find(params[:id])
+    @booking.user_id = current_user.id
+    @booking.furnishing_id = params[:furnishing_id]
+
+    if booking.save
+      redirect_do user_path
```
Comment (Chloebm) [inline app/controllers/bookings_controller.rb]: change to "to"

Comment (Chloebm) [reply app/controllers/bookings_controller.rb]: Add @user in user path

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -1,2 +1,50 @@
...
+  end 
+
+  def edit
+    @booking = Booking.find(params[:id])
+  end 
+  
+  # def update
+  #   @booking = Booking.find(params[:id])
+  #   if booking.update(booking_params)
+  #     redirect_to user_path notice: "Your booking was successfully added"
```
Comment (Chloebm) [inline app/controllers/bookings_controller.rb]: (@user)

File: app/controllers/bookings_controller.rb [hunk 4]
```diff
@@ -1,2 +1,50 @@
...
+  #   else
+  #     render :edit
+  #   end 
+  # end 
+
+  # def destroy
+  #   @booking = Booking.find(params[:id])
+  #   @booking.destroy
+  #   redirect_to user_path notice: "Your booking was successfully deleted"
```
Comment (Chloebm) [inline app/controllers/bookings_controller.rb]: (@user)

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (tim94white) [reply app/controllers/bookings_controller.rb]: nice, done

File: app/views/bookings/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<div class="mb-4 d-flex align-items-center justify-content-center py-5">
+  <div class="text-center text-white">
+    <h3>Add a booking to <%= @furnishing.title %></h3>
```
Comment (hebazmalik) [inline app/views/bookings/new.html.erb]: 'Book: @furnishing.title' might sound better

File: app/controllers/bookings_controller.rb (hunk 2 above)
Comment (tim94white) [reply app/controllers/bookings_controller.rb]: thanks

File: app/controllers/bookings_controller.rb (hunk 3 above)
Comment (tim94white) [reply app/controllers/bookings_controller.rb]: thanks

File: app/controllers/bookings_controller.rb (hunk 4 above)
Comment (tim94white) [reply app/controllers/bookings_controller.rb]: thanks

**Review by hebazmalik — APPROVED** [review]

File: app/views/bookings/new.html.erb (hunk 5 above)
Comment (tim94white) [reply app/views/bookings/new.html.erb]: thanks

**Pull request merged**